tester_app/client/app/bundles/HelloWorld/startup/clientGlobals.jsx

```diff
@@ -0,0 +1,3 @@
+import helloWorldClientEntryComponent from './helloWorldClientEntryComponent';
+
```

Comment:
// This is how react_on_rails can see the HelloWorldApp in the browser.
window.HelloWorldContainer = helloWorldContainer;